webserver/webserver/src/main/java/io/helidon/webserver/WebServer.java

```diff
@@ -143,6 +143,23 @@ default int port() {
      */
     int port(String socketName);
 
+    /**
+     * Update the TLS configuration of the default socket {@link WebServer#DEFAULT_SOCKET_NAME}.
+     *
+     * @param tls new TLS configuration
+     * @throws IllegalStateException if the new tls is not enabled or currently set tls is null
+     */
+    void updateTls(WebServerTls tls);
+
+    /**
+     * Update the TLS configuration of the named socket.
+     *
+     * @param tls new TLS configuration
+     * @param socketName specific named socket name
+     * @throws IllegalStateException if the new tls is not enabled or currently set tls is null
+     */
+    void updateTls(WebServerTls tls, String socketName);
```

Comment:
Should we update the prototype to also throw the exception ? I think checkstyle might complain.

Reply:
Hmmm I dont think it will. I have checked the other methods in this interface and they have it the same way.